events/app.py

```diff
@@ -82,12 +79,12 @@ def build_event_info(info, time):
     return {**info, 'created_at': time}
 
 
-def build_events_dict(events):
+def build_events_dict(events_cursor):
```

Comment:
Good call to name this a cursor to make it clear that the call may end up moving the underlying cursor.